conf/default.php

```diff
@@ -5,4 +5,4 @@
  * @author Andreas Gohr, Michael Große <dokuwiki@cosmocode.de>
  */
 
-//$conf['fixme']    = 'FIXME';
+$conf['dformat']    = 'Y/m/d';
```

Comment:
This config should be in the type's config similar to the format we have in the Integer type.